src/core/reactor_backend.cc

```diff
@@ -594,6 +603,7 @@ future<> reactor_backend_aio::readable_or_writeable(pollable_fd_state& fd) {
 
 void reactor_backend_aio::forget(pollable_fd_state& fd) noexcept {
     auto* pfd = static_cast<aio_pollable_fd_state*>(&fd);
+    pfd->abort();
```

Comment:
You missed reactor_backend_aio (maybe it didn't exist at the time of writing).

Maybe better to call from the caller?

Reply:
I added support for `reactor_backend_aio` and `reactor_backend_uring`